src/becca/becca-interface.ts

```diff
@@ -129,15 +129,18 @@ export default class Becca {
         return this.branches[branchId];
     }
 
-    getBranchOrThrow(branchId: string): BBranch | null {
+    getBranchOrThrow(branchId: string): BBranch {
         const branch = this.getBranch(branchId);
         if (!branch) {
             throw new NotFoundError(`Branch '${branchId}' was not found in becca.`);
         }
         return branch;
     }
 
-    getAttribute(attributeId: string): BAttribute | null {
+    getAttribute(attributeId: string | null): BAttribute | null {
```

Comment:
I think we shouldn't have a case where we call this method without an attributeId.

Reply:
@alexpietsch , you're right, I've pushed a fix.